build-system/tasks/storybook-react/react-env/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const path = require('path');
+const {BUILD_CONSTANTS} = require('../../../compile/build-constants');
+const {DefinePlugin} = require('webpack');
+const {getRelativeAliasMap} = require('../../../babel-config/import-resolver');
+const {webpackConfigNoChunkTilde} = require('../../storybook/env-utils');
+
+const rootDir = path.join(__dirname, '../../../..');
+
+module.exports = ({config}) => {
+  config.resolveLoader = {
+    modules: [
+      path.join(__dirname, '../node_modules'),
+      path.join(rootDir, 'node_modules'),
+    ],
+  };
+  config.resolve = {
+    modules: [
+      path.join(__dirname, '../node_modules'),
+      path.join(rootDir, 'node_modules'),
+    ],
+    alias: {
+      // Alias the accordion
+      '../../../amp-accordion/1.0/component':
+        '../../../amp-accordion/1.0/dist/component-react.max',
+      // Alias the base carousel
+      '../../../amp-base-carousel/1.0/component':
+        '../../../amp-base-carousel/1.0/dist/component-react.max',
+      // Alias the base carousel styles
+      '../../../amp-base-carousel/1.0/component.jss':
+        '../../../amp-base-carousel/1.0/dist/styles.css',
+      // Alias the component
+      '../component': '../dist/component-react.max',
+      // Alias the styles
+      '../component.jss': '../dist/styles.css',
+      ...getRelativeAliasMap(rootDir),
+      // Alias preact to react
+      '#preact': 'react',
+    },
+  };
+  config.module = {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        loader: 'babel-loader',
+        exclude: /node_modules/,
+        query: {
+          presets: [
+            [
+              '@babel/preset-env',
+              {
+                bugfixes: true,
+                targets: {'browsers': ['Last 2 versions']},
+              },
+            ],
+            [
+              '@babel/preset-react',
+              {
+                'runtime': 'automatic',
+              },
+            ],
```

Comment:
It's possible the linter won't allow this, but the amount of vertical space demands I at least try 😛 
```suggestion
            ['babel/preset-react', {'runtime': 'automatic'}],
```

Reply:
It appears to be working.